lxd/db/cluster/schema.go

```diff
@@ -276,30 +276,31 @@ CREATE VIEW instances_snapshots_devices_ref (
      JOIN instances ON instances.id=instances_snapshots.instance_id
      JOIN projects ON projects.id=instances.project_id
      JOIN instances_snapshots ON instances_snapshots.id=instances_snapshots_devices.instance_snapshot_id;
-CREATE TABLE networks (
+CREATE TABLE "networks" (
```

Comment:
Any idea what's going on with the sudden quoting?

Is that just what update-schema did?

Reply:
@stgraber yes I think so, I didn't do it manually.

Reply:
@stgraber seems its happened before, see the `config` and `images` tables in that file too.